tests/test_assignment.py

```diff
@@ -0,0 +1,91 @@
+import os
+import pytest
+
+import partridge as ptg
+
+import _fasttrips
+from fasttrips import Trip
+from fasttrips import Util
+
+HOME_DIR = os.path.join(os.getcwd(), "fasttrips", "Examples",)
+TEST_HOME_DIR = os.path.join(HOME_DIR, 'test_assignment')
+OUTPUT_DIR = os.path.join(TEST_HOME_DIR, 'output')
+
+@pytest.mark.skip(reason='Need to upgrade to access a non-static method')
```

Comment:
?

Reply:
This test is a little messy, and not fully implemented. Attempting to identify the bare minimum necessary to run the C++ wrapper code and isolate assignment.